schema.json

```diff
@@ -0,0 +1,23 @@
+{
+    "title": "event",
+    "type": "object",
+    "properties": {
+ 	 "organizer": {
+        "name": "name",
+        "email": "email"
+  	 },
+  	"id": "number",
+    "name": "event_name",
```

Comment:
I assumed "title" and "name" are the same thing, so I didn't add something for "name"... unless it was something else?